cdap-master/src/main/java/co/cask/cdap/data/tools/ReplicationStatusTool.java

```diff
@@ -67,13 +77,25 @@
   private static Long masterShutdownTime = 0L;
   private static String inputStatusFileName = null;
   private static String outputStatusFileName = null;
+  private static String dirsFileName = null;
   private static boolean debugDump = false;
-  private static String delimiter = ":";
+  private static String delimiter = "#";
+  private static String checksumsEnd = "EndofChecksums";
+  private static String tablesEnd = "EndofTables";
+
+  // Default paths to be pulled from configuration
+  private static final String[] ALL_CHECKSUM_PATHS_DEFAULT = {
+    "data.tx.snapshot.dir" // this is only defined at tephra/TxConstants.java
```

Comment:
I was wondering if we wanted to package tephra dependencies into the tool for the one constant. Adding it for now.